File: template.cpp
#include <iostream>
#include <fstream>
#include <random>
#include <algorithm>

using namespace std;

#define int long long

int randint(int start, int end) {
    random_device rd;
    mt19937 gen(rd());
    uniform_int_distribution<int> dis(start, end);
    return dis(gen);
}

string to_4digit(int n) {
    string s = to_string(n);
    while (s.size() < 4) {
        s = "0" + s;
    }
    return s;
}

signed main() {

    // check tests folder exists or not, if not, create it
    system("if [ ! -d tests ]; then mkdir tests; fi");
    // A + B example
    int from = 1;
    int to = 100;
    for (int i = from; i <= to; i++) {
        int A = randint(1, 1e9);
        int B = randint(1, 1e9);

        ofstream f;
        f.open("tests/" + to_4digit(i) + ".in");
        // input here
        f << A << " " << B << endl;
        f.close();
        f.open("tests/" +  to_4digit(i) + ".out");
        // output here
        f << A + B << endl;
        f.close();
    }
}

File: template.py
import os
from random import randint


try:
    os.mkdir('tests')
except FileExistsError:
    pass


start = 1
end = 100
for i in range(start, end + 1):
    A = randint(1, int(1e9))
    B = randint(1, int(1e9))
    with open(f'tests/{i:04}.in', 'w') as f:
        # input here
        f.write(f'{A} {B}')
    with open(f'tests/{i:04}.out', 'w') as f:
        # output here
        f.write(f'{A + B}')
